terraform/datadog/etcd.tf

```diff
@@ -0,0 +1,23 @@
+resource "datadog_monitor" "etcd_one_leader" {
+  name                = "${format("%s etcd IsLeader count is exactly one (%s)", var.env, element(list("upper", "lower"), count.index))}"
+  type                = "query alert"
+  message             = "There is not exactly one etcd reporting IsLeader."
+  escalation_message  = "There is still not exactly one etcd reporting IsLeader."
+  notify_no_data      = false
+  require_full_window = true
+
+  count = 2
+
+  query = "${
+    format(
+      "min(last_5m):sum:cf.etcd.IsLeader{deployment:%s}.fill(last, 60) %s",
+      var.env,
+      element(list("> 1", "< 1"), count.index)
+    )}"
+
+  thresholds {
+    critical = "${element(list("1", "1"), count.index)}"
```

Comment:
OK, I changed it and I put a warning.

But funny thing, we are already thinking to set different limits, as using "min" might be too sensitive. as commented in slack:

https://govuk.slack.com/archives/the-government-paas/p1486042172009010
> During a rolling deployment of CF, the metric might be reported twice or we might miss metrics, so the sum across the members might get 2 or 0 for one value.

> Using min, the monitor is sensible to restarts and can alert for 5m during a rolling deployment. 

> one way to avoid this could be use "avg" instead of "min" and a lower/upper limit of 0.80 and 1.20, for instance. As the metric is reported every 1 minute, the average in 5 minutes would be within 0.9-1.10